<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Device">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>331</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>HeartWave Device</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="Screen" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>291</width>
       <height>171</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QCustomPlot" name="Graph" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>241</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="BreathPacer" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 250, 185)</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="BatteryBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>5</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Sensor">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="CoherenceLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>coherence</string>
      </property>
     </widget>
     <widget class="QLabel" name="LengthLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>length</string>
      </property>
     </widget>
     <widget class="QLabel" name="AchievementLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>achievement</string>
      </property>
     </widget>
     <widget class="QLabel" name="CoherenceVal">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="LengthVal">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="AchievementVal">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="SummaryView" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLabel" name="CoherenceLevels">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="MetricsLabel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="LogHistoryMetrics" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>241</width>
        <height>121</height>
       </rect>
      </property>
      <widget class="QLabel" name="CoherenceLevels_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="MetricsLabel_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="LogHistoryMetricsLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="MenuButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UpButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DownButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RightButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>310</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LeftButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SelectButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PowerButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>I/O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QLabel" name="RedLight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="BlueLight">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="GreenLight">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>20</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>16</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="SensorScenario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Heart Sensor Monitor Scenario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LowBatteryScenario">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Battery Scenario</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
